src/components/Calculator/index.jsx

```diff
@@ -1,11 +1,25 @@
 import React from 'react';
 import Display from '../Display';
 import Number from '../Number';
-import OperationEditor from '../OperationEditor';
+import Operator from '../Operator';
+import Result from '../Result';
 
 const Calculator = () => {
     const[currentNumber, setCurrentNumber] = React.useState('');
-    
+    const[currentOperation, setCurrentOperation] = React.useState([]);
```

Comment:
nit

Seems like if you are using an array, it means a list of elements, so the name of the var could fit better as `currentOperations`

Reply:
ok